climlab/tests/test_advdiff_procs.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import division
+import numpy as np
+import climlab
+import pytest
+
```

Comment:
```suggestion
from climlab import constants as const
```
should fix the errors currently showing up in the test suite.